### ISSUE TITLE
Add limit for tag name - 30

### ISSUE THREAD

**Issue by tritonJS826** [tritonJS826/masters-way#1102]

**Comment by tritonJS826** [comment]
done on the back

**Comment by Ekaterina1994** [comment]
Diplicate of #973 / Need more details - need limit for tag or labels and etc.

**Comment by tritonJS826** [comment]
Yeah right - need visual limit on the frontend 30 chars for label in the modal

**Comment by tritonJS826** [comment]
duplicate of #973

**Comment by EkaterinaSokolova-V** [comment]
It is impossible to create a tag with a name longer than 30 characters. 
However, it is necessary to inform the user that they have entered an unsupported number of characters while assigning the name. Currently, any number of characters can be entered, and only after clicking 'OK' does the error message appear.

**Comment by Ekaterina1994** [comment]
@EkaterinaSokolova-V As was mentioned it's a duplicate of #973. Should be closed
